packages/components/src/calendar/stories/date-picker.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+import { withState } from '@wordpress/compose';
+import { DatePicker, H, Section } from '@woocommerce/components';
+
+const dateFormat = 'MM/DD/YYYY';
+
+const DatePickerExample = withState( {
```

Comment:
These withState calls unfortunately make Storybook incapable of determining the props for tweaking. Stories like this will need work too.